rerun_py/rerun_sdk/rerun/__main__.py

```diff
@@ -2,19 +2,19 @@
 
 from __future__ import annotations
 
-import rerun_bindings as bindings
+import os
+import subprocess
+import sys
 
-from rerun import unregister_shutdown
 
+def main() -> int:
```

Comment:
So running `python -m rerun-sdk` will still work? But we don't need to run the `unregister_shutdown` anymore?

Reply:
Good catch -- I had done some refactoring that kept us from needing it but then moved stuff around so we actually still benefit from it.